connectors/src/api/slack_channels_linked_with_agent.ts

```diff
@@ -98,6 +120,27 @@ const _patchSlackChannelsLinkedWithAgentHandler = async (
       )
     );
   });
+  const joinPromises = await Promise.all(
+    slackChannelIds.map((slackChannelId) =>
+      joinChannel(parseInt(connectorId), slackChannelId)
```

Comment:
So the Bot joins automatically a channel if the channel is selected in the list to be able to answer but it won't index the content of this channel?

Reply:
Correct.